src/redisgears_python.c

```diff
@@ -30,6 +30,10 @@ static RedisModuleCtx* currentCtx = NULL;
 static bool blockingExecute = true;
```

Comment:
all those vars should be move to thread local, this might break if we create an execution from another thread while also creating from the main thread (like for example, an execution that creates execution inside it). We will do it in another PR, lets just add a todo here

Reply:
Sure.

Reply:
Resolved via a "TODO"